helpers/press-release-sparql-queries.js

```diff
@@ -138,3 +154,52 @@ export async function createPublicationTasksPerPublicationChannel(graph, pressRe
 		throw err;
 	}
 }
+
+export function getPublicationChannelsByPublicationEvent(graph, publicationEvent) {
+	return query(`
+		${PREFIXES}
+		
+		SELECT ?pubChannel
+		WHERE {
+			GRAPH ${sparqlEscapeUri(graph)} {
+				${sparqlEscapeUri(publicationEvent)}	ext:publicationChannels 	?pubChannel .
+			}
+		}
+	`);
+
+}
+
+export async function getPlannedPublicationEvents() {
+	const now = new Date();
+	const queryResult = await query(`
+			${PREFIXES}
+			
+			SELECT ?graph ?publicationEvent
+			 WHERE {
+				GRAPH ?graph {
+					{
+						?publicationEvent		a		ebucore:PublicationEvent .
+						FILTER NOT EXISTS { ?publicationEvent 	ebucore:publicationStartDateTime 	?publicationStartDateTime }
+					}
+					UNION	
```

Comment:
`UNION` is niet correct in deze query. Nu maak je de verzameling van alle publication-events zonder startdatum en alle publication-events met een geplande startdatum in het verleden, terwijl je enkel de publication-events wil die aan beide voorwaarden voldoen.